src/main/web_src/src/service/SelectOptions.tsx

```diff
@@ -200,6 +200,58 @@ const selectOptions: SelectOptions = {
 		{ value: 'MIDLERTIDIG', label: 'Midlertidig' }
 	],
 
+	ovrigIkkeOppholdsKategoriArsak: [
+		{ value: 'OPPHEVET_INNREISEFORBUD', label: 'Opphevet innreiseforbud' },
+		{ value: 'ANNULERING_AV_VISUM', label: 'Annulering av visum' },
+		{ value: 'UTLOPT_OPPHOLDSTILLATELSE', label: 'Utløpt oppholdstillatelse' },
+		{
+			value: 'UTLOPT_EO_SELLER_EFTA_OPPHOLDSRETT_ELLER_EO_SELLER_EFTA_OPPHOLDSTILLATELSE',
+			label: 'Utløpt oppholdsrett/oppholdstillatelse'
+		}
+	],
+
+	avslagGrunnlagOverig: [
+		{ value: 'PERMANENT', label: 'Permanent' },
+		{ value: 'BESKYTTELSE', label: 'Beskyttelse' },
+		{
+			value: 'STERKE_MENNESKELIGE_HENSYN_ELLER_SAERLIG_TILKNYTNING_TIL_NORGE',
+			label: 'Sterke menneskelige hensyn'
+		},
+		{ value: 'ARBEID', label: 'Arbeid' },
+		{ value: 'UTDANNING', label: 'Utdanning' },
+		{ value: 'FAMILIE', label: 'Familie' },
+		{ value: 'SELVSTENDIG_NAERINGSVIRKSOMHET', label: 'Selvstendig næringsvirksomhet' },
+		{ value: 'ANNET', label: 'Annet' },
+		{ value: 'UAVKLART', label: 'Uavklart' }
+	],
+
+	avslagGrunnlagTillatelseGrunnlagEOS: [
+		{
+			value: 'EGNE_MIDLER_ELLER_FASTE_PERIODISKE_YTELSER',
+			label: 'Egne Midler/Faste ytelser'
```

Comment:
Små bokstaver på alt annet enn første bokstav? 😁  Gjelder for noen flere tilfeller nedover her også.

Reply:
Bra sett, skal fikse 😄